go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mattermost/mattermost-load-test-ng
 
-go 1.22
+go 1.22.0
```

Comment:
Do we need to specify the patch version here? Did `go mod tidy` change this?

Reply:
yes, this was modified by `go mod tidy` but maybe it happened cause I'm running go 1.22.0 locally ?

Reply:
Just tested locally with 1.22.6 and it sets it to 1.22.0 as well. Not sure what the need there is, but let's keep it!